starknet_devnet/util.py

```diff
@@ -246,7 +246,6 @@ def generate_state_update(previous_state: CarriedState, current_state: CarriedSt
 DEFAULT_GENERAL_CONFIG = StarknetGeneralConfig.load({
     "event_commitment_tree_height": 64,
     "global_state_commitment_tree_height": 251,
-    'gas_price': 100000000000,
     'starknet_os_config': {
         'chain_id': 'TESTNET',
         'fee_token_address': '0x20abcf49dad3e9813d65bf1b8d54c5a0c9e6049a3027bd8c2ab315475c0a5c1'
```

Comment:
check if this can be improved as discussed

Reply:
[b5192b7](https://github.com/Shard-Labs/starknet-devnet/pull/70/commits/b5192b7ae38633593909e81689b052ac478c4b0f) and [b5192b7](https://github.com/Shard-Labs/starknet-devnet/pull/70/commits/b5192b7ae38633593909e81689b052ac478c4b0f)